crates/rethnet/src/config.rs

```diff
@@ -34,10 +34,14 @@ pub const DEFAULT_PRIVATE_KEYS: [&str; 20] = [
 pub struct ConfigFile {
     // TODO: expand this per https://github.com/NomicFoundation/rethnet/issues/111
     pub accounts: Vec<AccountConfig>,
+    pub chain_id: U64,
```

Comment:
What's the reason you're using a `U64` instead of a `u64` for the config file?

Reply:
Good question. I guess I was just being consistent everywhere, but clearly it makes sense for this to be a `u64`, so I did that in https://github.com/NomicFoundation/hardhat/pull/4209/commits/26610b346e327832760e47491b4df78f3e8cb8a6